ONSdigital/ras-rm-performance-tests
# added json response file

# What and why?
The performance tests needed to make use of a JSON file when handling requests so that it is easier to add more requests to the tests in the future. It also makes the tests click on the history and secure message tabs on the frontstage surveys page.

# How to test?
Run the Locust tests locally. This is best done using [Mark's branch](https://github.com/ONSdigital/ras-rm-performance-tests/pull/26). Note that you'll have to copy over the `locustfile.py` and the `requests.json` files from this branch to that one. Also, you might need to export the value for the `requests_filepath` variable in `values.yaml` and change the filepath at line 31 in locustfile.py to begin with `.//` instead of `/mnt/locust/`.

You can also try a helm deployment of this PR and make sure the file is being read in okay.

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-1065)
